### PR TITLE
Request: Clockify Icon

### PR DESCRIPTION
**Pull request by malizmaj**
https://clockify.me

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -271,6 +271,10 @@
             "source": "https://circleci.com/press"
         },
         {
+            "title": "Clockify",
+            "hex": "03A9F4",
+            "source": "https://clockify.me/"
+        },        {
```
Comment (birjolaxew) [inline _data/simple-icons.json]: Could you make sure that the formatting matches the rest of the file? The closing curly bracket should have a newline after the comma.

File: icons/clockify.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-clockify-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-clockify-icon">Clockify icon</title><path fill-rule="evenodd" clip-rule="evenodd" d="M14.16 8.552l5.508-5.514 1.577 1.577-5.508 5.514-1.577-1.577zm-2.231 5.635a1.956 1.956 0 0 1-1.951-1.961c0-1.083.873-1.961 1.95-1.961 1.078 0 1.952.878 1.952 1.96a1.956 1.956 0 0 1-1.951 1.962zm9.341 5.695l-1.576 1.578-5.508-5.514 1.577-1.578 5.507 5.514z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M11.986 20.706c1.11 0 2.168-.221 3.137-.615l2.69 2.694a11.912 11.912 0 0 1-5.827 1.518C5.366 24.303 0 18.93 0 12.303s5.367-12 11.986-12c2.095 0 4.063.54 5.776 1.486l-2.645 2.65a8.323 8.323 0 0 0-3.131-.614c-4.636 0-8.394 3.78-8.394 8.441 0 4.661 3.758 8.44 8.394 8.44z"/></svg>
```
Comment (birjolaxew) [inline icons/clockify.svg]: A few things regarding the SVG:

- The `fill-rule` and `clip-rule` attributes on the `<path>`s can be safely removed in this case.
- The icon isn't centered in the SVG; the lower part of the circle is cut off, and there is extra space on the top and right of the SVG.

**Comment by malizmaj** [comment]
Fixed, thanks for the feedback :)

**Comment by ericcornelissen** [comment]
@malizmaj we try to have icons fill up as much space as possible, which currently is not the case as you can see ⬇️. Could you update the SVG so that the icon touches at least two of the borders?

image

**Comment by malizmaj** [comment]
Sure, np :)

**Comment by birjolaxew** [comment]
:+1: Thanks for the contribution! It is much appreciated.

**Pull request merged**